package.json

```diff
@@ -29,40 +29,44 @@
       "url": "https://technote.space"
     }
   ],
-  "main": "build/index.js",
-  "types": "build/index.d.ts",
+  "type": "module",
+  "exports": "./dist/index.mjs",
```

Comment:
specifically, adding "exports" is ALWAYS a breaking change, and this particular form makes the package ESM-only. This part of v2.5 must be reverted, and published as a v2.5.1.